OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The various different versions of Visual Studio and Monodevelop/Xamarin Studio have different conventions with regard to the BOM, trailing newlines, and general project layout.  Our general policy is that individual prs should only commit real changes to cut down on noise as these other things are repeatedly changed back and forward.